### ISSUE TITLE
Loading spinner not centered

### ISSUE THREAD

**Issue by nshindarev** [telekom/scale#1740]
**Scale Version**
^3.0.0-beta.117

**Framework and version**
Vue.js: ^3.2.45

**Current Behavior**
Loading spinner is not centered:
![image](https://user-images.githubusercontent.com/9587600/233052894-308eb728-49fe-4e25-b7b9-28f840e7f6bd.png)

HTML element added without any additional changes:
```HTML
<scale-loading-spinner
          v-if="loading"
          size="large"
          text="Loading..."
          alignment="vertical"
        ></scale-loading-spinner>
```

**Expected Behavior**
Loading spinner is centered relatively to parent div and to "...Loading" inner text
![image](https://user-images.githubusercontent.com/9587600/233054237-c5f75dc0-ac70-4ceb-8787-c4a11bb453a1.png)

**Code Reproduction**

https://codesandbox.io/s/scale-loading-spinner-issue-p4k58p

**Desktop (please complete the following information):**
 - OS: macOS Ventura
 - Browser [mozilla, chrome, safari]
 - Version [mozilla: 112.0.1 (64-bit)]

**Temporare fix in our project:**
``` HTML 
<style scoped>
scale-button {
  display: block;
  margin-top: 12px;
}

scale-loading-spinner::part(container){
  padding-right: 55px;
  padding-top: 10px;
  padding-bottom: 5px;
  text-align: center;
}
</style>
```

**Comment by ayeshaa-m** [comment]
@nshindarev The fix will be coming in upcoming release. Thanks

**Comment by nshindarev** [comment]
Thanks a lot!
@ayeshaa-m  can you please tell in which release version it will be available?

**Comment by ayeshaa-m** [comment]
@nshindarev It will be available in version v3.0.0-beta.135
